pkg/bucket/bucket.go

```diff
@@ -56,6 +57,8 @@ type (
 	Option func(*bucketOptions)
 )
 
+var HTTPClient = &http.Client{}
```

Comment:
don't set global variable, pass it instead?

Reply:
the other way is to pass it in the cli context, what do you suggest?

Reply:
or as an optional value, and the constructor can check if it is nil or not, and sets it accordingly?